consensus/module.go

```diff
@@ -317,6 +318,22 @@ func (m *consensusModule) loadPersistedState() error {
 	return nil
 }
 
-func (m *consensusModule) EnableServerMode() {
-	m.stateSync.EnableServerMode()
+func (m *consensusModule) IsSynched() (bool, error) {
```

Comment:
In the interface, add a comment what this does: `Compares the local persisted state against the aggregated metadata from peers. There is never a 100% guarantee that it's synched`.

In the codebase, just say `IsSynched implements the interface function for ...`. 

Bryan's PR in #558 follows a goo pattern for this.

Reply:
Thanks!